src/main/java/session_17_equals_hashcode_generics/challenge/exercise_18/ConvertXmlToJson.java

```diff
@@ -0,0 +1,22 @@
+package session_17_equals_hashcode_generics.challenge.exercise_18;
+
+import org.json.JSONObject;
+import org.json.XML;
+
+import java.io.File;
+import java.nio.file.Files;
+
+public class ConvertXmlToJson {
+
+    public static void main(String[] args) throws Exception {
+        File xmlFile = new File("C:\\Personal\\IT School\\src\\main\\java\\session_17_equals_hashcode_generics\\challenge\\exercise_18\\bookstore.xml");
```

Comment:
We can maybe declare the path to the XML file as a constant or configurable variable? Will this have the potential to improve readability?
`private static final String XML_FILE_PATH = "C:\\Personal\\IT Scho...`